internal/node/model/models.go

```diff
@@ -11,10 +11,11 @@ import (
 type (
```

Comment:
This is an exception, because we are reexporting these names.
Leaving these alias as is is fine.